model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -203,6 +203,23 @@ message PCollection {
   DisplayData display_data = 5;
 }
 
+// The payload for an executable stage. This will eventually be passed to an SDK in the form of a
+// ProcessBundleDescriptor.
+message ExecutableStagePayload {
+
+  Environment environment = 1;
```

Comment:
I'd like a comment explaining why this is a value rather than an ID, or replace it with `string environment_id`

Reply:
Added a comment.